metricflow-semantics/tests_metricflow_semantics/snapshots/test_linkable_spec_resolver.py/str/test_create_linkable_element_set_from_join_path_multi_hop__result0.txt

```diff
@@ -1,48 +1,48 @@
-Model Join-Path                                  Entity Links          Name             Time Granularity    Date Part    Properties
```

Comment:
When looping through models in the join path, I added a new assertion that the model exists in `SemanticModelLookup`. This assertion caused the test to break. This is expected since the model `bookings` doesn't exist. The file names it `bookings_source`, so I changed the test comparison to expect `bookings_source` instead.